src/store/participantsStore.js

```diff
@@ -635,6 +694,14 @@ const actions = {
 		}
 		context.commit('updateParticipant', { token, attendeeId, updatedData })
 	},
+
+	async setTyping(context, { typing }) {
+		if (!context.getters.currentConversationIsJoined()) {
```

Comment:
`context.getters.currentConversationIsJoined` is not a function here. Maybe we could fix it here:
https://github.com/nextcloud/spreed/blob/b476620d2eb3d8fa7126b93225f8904950be9442/src/store/tokenStore.js#LL47C1-L49C4

```js
	currentConversationIsJoined: (state) => () => {
		return state.lastJoinedConversationToken === state.token
	},
```

Reply:
It was a typo in `setTyping`, it should have been a plain value, not a function :-) Well, it would make sense if it was a function, but it is used as a plain value elsewhere in the code, so I kept that.

Nevertheless I fixed the getter, as it was not using the given state but the variable defined above.